utils/check_symbols.sh

```diff
@@ -39,7 +38,7 @@ for line in `nm -P -u "${NDPI_LIB}"`; do
                     'malloc'|'calloc'|'realloc'|'free') SKIP=1 ;;
                 esac
             ;;
-            '[ndpi_utils.o]'|'[ndpi_memory.o]'|'[roaring.o]')
+            '[ndpi_utils.o]'|'[ndpi_memory.o]')
```

Comment:
@pasabanov, could you explain why checking  `roaring.o` is no more necessary, please?

Reply:
Because there is the same pattern before on line 37 (36).

Reply:
yeah, you are right. Thanks!